packages/terra-overlay/src/Overlay.jsx

```diff
@@ -125,21 +133,24 @@ class Overlay extends React.Component {
     }
   }
 
-  resetBeforeOverlay() {
+  resetBeforeOverlay(selector) {
     document.documentElement.style.overflow = this.overflow;
     if (this.props.isRelativeToContainer) {
       this.enableContainerChildrenFocus();
     }
+    if (document.querySelector(selector)) { // null on initial page load
+      document.querySelector(selector).removeAttribute('aria-hidden');
+    }
   }
 
   render() {
     const {
-      children, isOpen, backgroundStyle, isScrollable, isRelativeToContainer, onRequestClose, ...customProps
+      children, isOpen, backgroundStyle, isScrollable, isRelativeToContainer, onRequestClose, rootSelector, ...customProps
     } = this.props;
     const type = isRelativeToContainer ? 'container' : 'fullscreen';
 
     if (!isOpen) {
-      this.resetBeforeOverlay();
+      this.resetBeforeOverlay(this.props.rootSelector);
```

Comment:
we can remove `this.props` and just use `rootSelector` - we have already unpacked `this.props` into distinct prop variables on line 148

Reply:
Change made!